api/app/signals/apps/api/v1/urls.py

```diff
@@ -28,6 +30,8 @@
 # Public API
 public_router = SignalsRouterVersion1()
 public_router.register(r'public/signals', PublicSignalViewSet, basename='public-signals')
+if settings.ENABLE_PUBLIC_GEO_SIGNAL_ENDPOINT:
```

Comment:
Personally I would like to see this line at the end of registering routes. I think it reads better that way.

Like:
```
# Public API
public_router = SignalsRouterVersion1()
public_router.register(r'public/signals', PublicSignalViewSet, basename='public-signals')
public_router.register(r'public/feedback/standard_answers', StandardAnswerViewSet, basename='feedback-standard-answers')
public_router.register(r'public/feedback/forms', FeedbackViewSet, basename='feedback-forms')
public_router.register(r'public/areas', PublicAreasViewSet, basename='public-areas')

public_categories = public_router.register(r'public/terms/categories', PublicCategoryViewSet,
                                           basename='public-maincategory')
public_categories.register(r'sub_categories', PublicCategoryViewSet, basename='public-subcategory',
                           parents_query_lookups=['parent__slug'])

if settings.ENABLE_PUBLIC_GEO_SIGNAL_ENDPOINT:
    public_router.register(r'public/map-signals', PublicSignalListViewSet, basename='public-list-signals')

# Private API
```

Reply:
check